src/js/components/editor/Column.vue

```diff
@@ -0,0 +1,201 @@
+<template>
+    <div class="editor-column">
+        <div class="text-col inline text-sbl-hebrew" dir="rtl" contenteditable="true" @keydown="onKeydown" @input="onInput" @paste="onPaste" @copy="onCopy" v-html="colHtmlString">
```

Comment:
@Bronson-Brown-deVost : I'd like to discuss this approach. I think it's a good one so far. By using "contenteditable", we can rely on the browser for the following things:

1. text input (including multi-language support), adding lines, deleting lines.
2. Navigating via arrow keys
3. History: ctl-z ctl-y style.
4. Handling normal text-editing hot-keys

This leads to a fairly fast performance for the editing experience. Then, the model is synchronized to the DOM.

Whereas we don't have to roll our own implementation for all the above, there's a bit more work to synchronize model <> view. My sense thus far is that this is _simpler_ than having to implement all those features manually.

Effectively, the model becomes _reactive_ to the DOM state, updating on DOM events.

Reply:
This sounds nice.  I had played around once with contenteditable and I forget why I did not use it, but perhaps you have already overcome whatever problem I had encountered or had simply imagined.  Let's give it shot and see how it goes, I like the idea of relying on the standard browser functionality whenever possible.

Reply:
Ok, sounds good. Currently, my implementation is only able to handle adding new lines by hitting enter. I will get a little bit further with a limited set of functionality and then request review (and get all tests patched up). The current implementation allows editing of the text, but the model isn't synchronized so we can't really track changes for most actions.

Then will probably break out various pieces of functionality and then start picking them off (e.g., implement history, implement copy/paste, implement font changes, adding columns).